understanding/20/non-text-content.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Non-text Content</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Non-text information is available to more people</dd>
+		<dt>Author task</dt><dd>Create a text alternative for visual and auditory content</dd>
+		<dt>Key beneficiaries</dt><dd>Users with reduced sight, hearing, or cognition</dd>
```

Comment:
@gundulaniemann said:
> I understand from the SC that the text alternative might be available only to assistive technology.
As a consequence, only users using assistive technology benefit.
Suggestion:
Benefeciaries: Users using assistive technology

Not all users would need to use assistive technology to derive benefit from the techniques. I believe the current wording is better; however, as previously stated, planning to ditch the Key beneficiaries line